core/primitives/res/runtime_configs/42.txt

```diff
@@ -0,0 +1 @@
+storage_amount_per_byte: 10000000000000000000
```

Comment:
It would be great to have a test that would verify that or all of these configs end up the same as the data in legacy configs.

Reply:
I think that's `test_runtime_config_data`? It would be great if we can re-use that test without actually continuing to carry JSONs. I think hashes today are actually hashes of JSON, but we can probably use StableHash of the RuntimeConfig struct itself? If we *first* change the test, and than add parameters, we'll check that hashes are the same.